modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerIndexForceRebuildTest.java

```diff
@@ -279,7 +293,24 @@ public void testGroupNamesArg() throws Exception {
 
             waitForIndexesRebuild(grid(LAST_NODE_NUM));
 
-            validateTestCacheGroupArgOutput();
+            String outputStr = testOut.toString();
+
+            validateOutputCacheGroupsNotFound(
+                outputStr,
+                ImmutableMultimap.of(GRP_NAME_NON_EXISTING, CACHE_NAME_NON_EXISTING)
+            );
+
+            validateOutputIndicesRebuildingInProgress(outputStr, ImmutableMultimap.of(GRP_NAME_1, CACHE_NAME_1_2));
```

Comment:
You can just use a java.util.Collections#singletonMap or F.asMap(K, V, K, V)

Reply:
Unfortunately, java.util.Map and com.google.common.collect.Multimap not compatible